BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -98,21 +107,32 @@ void BSP_CAN_Init(void) {
     NVIC_InitStructure.NVIC_IRQChannelSubPriority = 0x0;
     NVIC_InitStructure.NVIC_IRQChannelCmd = ENABLE;
     NVIC_Init(&NVIC_InitStructure);	
+
+
+    //return CANinitSuccess;
+    
 }
 
 /**
  * @brief   Transmits the data onto the CAN bus with the specified id
- * @param   id : Message of ID. Also indicates the priority of message. The lower the value, the higher the priority.
- * @param   data : data to be transmitted. The max is 8 bytes.
- * @param   length : num of bytes of data to be transmitted. This must be <= 8 bytes or else the rest of the message is dropped.
- * @return  0 if module was unable to transmit the data onto the CAN bus. Any other value indicates data was transmitted.
+* @param   id : Message of ID. Also indicates the priority of message. The lower the value, the higher the priority.
+* @param   Voltage : the voltage value at that moment in a float
+* @return  0 if module was unable to transmit the data onto the CAN bus. Any other value indicates data was transmitted.
  */
-uint8_t BSP_CAN_Write(uint32_t id, uint8_t data[8], uint8_t length) {
+uint8_t BSP_CAN_Write(uint32_t id, int32_t Voltage) {
+    
     TxMessage.StdId = id;
-    TxMessage.DLC = length;
-	for(int i = 0; i < length; i++){
-        TxMessage.Data[i] = data[i];
-	}
+       //floatTo4Bytes(Voltage, &TxMessage.Data[4]);
```

Comment:
Can we remove the commented out code in this function?

Reply:
Deleted that and some other commented out stuff in the BPS_CAN_Write function.